rioting-daisies/BangazonWorkforce
# Details and Edit for TrainingPrograms

# Description

This pull request contains the code that allows the user to view details of a specific training program. The details view accepts a newly created view model (trainingProgramDetailsView). Also, the edit view and functionality for training programs has been implemented.

Fixes Issue # [(11)](https://github.com/rioting-daisies/BangazonWorkforce/issues/11)

## Type of change

Please delete options that are not relevant.




# Testing Instructions

1. `git fetch --all`
2. If you've already checked out this branch, `git pull origin blue-DeleteTraining`
3. Review the code within the `TrainingProgramsController.cs`, `TrainingProgramDetailsViewModel.cs`,  `Views/TrainingPrograms/Details.cshtml`, and  `Views/TrainingPrograms/Edit.cshtml`
4. Click on the details of a training program in the index view, make sure the details show the employees signed up for the training program.
5. Click on edit for a training program and confirm that the functionality works.



# Checklist:
